### DISCUSSION TITLE
Filtering feature on map screen.

### DISCUSSION THREAD

**Discussion by teomanofficial** [acikkaynak/deprem-yardim-frontend#485] [Ideas]
Source: https://github.com/acikkaynak/deprem-yardim-frontend/issues/471

- We need filtering for the map screen, Not just date but other additional fields as well.
- Instead of picking date time specifically, intervals like an hour, last 3 hours etc.. would be much more practical.
- Additionally, based on the variables in the data (which provided by users) we can add more filters

**Comment by khnbzkrt** [comment]
I think the timezone change should also be made for the datetime values. I think it would be an advantage for our citizens living abroad.

**Reply by goktugs** [reply]
Adding time zone update option might make sense. UTC +3 Being default, then dropdown city selector.
If it can be resolved by BackEnd I'm on it.

**Comment by cdagli** [comment]
Duplicate https://github.com/acikkaynak/deprem-yardim-frontend/discussions/552

**Comment by handekaratas96** [comment]
@cdagli Yes, it's already been mentioned, adding time zone might be useful in long term but I think it is not urgent.
